docs/photoniq/p3/view-p3-audit-log.md

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_position: 60
+title: View P3 Audit Log
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+After you have created one or more policies, you can view the P3 audit log on the dashboard.
```

Comment:
What all is logged? We should tell the user what this log tracks.

Is it related to the Alerts at all? Like, will an alert move to the audit log once it is fixed?